src/modules/market/containers/market-view.js

```diff
@@ -18,27 +20,18 @@
 // //
 // // import getValue from 'utils/get-value'
 //
-// const mapStateToProps = state => ({
-//   // isConnected: state.connection.isConnected,
-//   // universe: state.universe,
-//   // isLogged: !!getValue(state, 'loginAccount.address'),
-//   // market: selectSelectedMarket(state),
-//   // orderCancellation: getOrderCancellation(),
-//   // numPendingReports: selectMarketsTotals(state).numPendingReports,
-//   // scalarShareDenomination: getScalarShareDenomination(),
-//   // closePositionStatus: selectClosePositionStatus(state),
-//   // marketDataNavItems: MARKET_DATA_NAV_ITEMS,
-//   // marketUserDataNavItems: MARKET_USER_DATA_NAV_ITEMS,
-//   // outcomeTradeNavItems: OUTCOME_TRADE_NAV_ITEMS
-// })
-//
-// const mapDispatchToProps = dispatch => ({
-//   // loadFullMarket: marketId => dispatch(loadFullMarket(marketId)),
-//   // updateSelectedMarketID: marketId => dispatch(updateSelectedMarketID(marketId)),
-//   // clearSelectedMarketID: () => dispatch(clearSelectedMarketID()),
-//   // toggleFavorite: marketID => dispatch(toggleFavorite(marketID))
-// })
-//
-// // const Market = withRouter(connect(null, null)(MarketView))
-//
-// export default Market
+const mapStateToProps = state => ({
+  isLogged: state.isLogged,
+  isConnected: state.connection.isConnected,
+  universe: state.universe,
+  marketsData: state.marketsData
```

Comment:
Since we don’t actually use `marketsData` in this view, I would personally use mergeProps, derive the marketID from the location, then pass down something like `isMarketLoaded`, which has it’s value as `state.marketsData[marketID] != null`.

Reply:
This now happens in mergeProps